### PR TITLE
ref(draft): remove ingress requirement

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

File: cmd/draft/init.go [hunk 1]
```diff
@@ -113,10 +118,9 @@ func (i *initCmd) run() error {
 
 		if cloudProvider != "" {
 			fmt.Fprintf(i.out, "\nDraft detected that you are using %s as your cloud provider. AWESOME!\n", cloudProvider)
-			fmt.Fprintf(i.out, "Draft will be using the following configuration:\n\n'''\n%s'''\n\n", chartConfig.GetRaw())
 
 			if !i.autoAccept {
-				fmt.Fprint(i.out, "Is this okay? [Y/n] ")
+				fmt.Fprint(i.out, "Is it okay to use the registry addon in minikube to store your application images? If not, we will prompt you for informaiton on the registry you'd like to push your application images to during development. [Y/n] ")
```
Comment (bacongobbler) [inline cmd/draft/init.go]: typo: information

File: cmd/draft/init.go [hunk 2]
```diff
@@ -76,6 +80,7 @@ func newInitCmd(out io.Writer, in io.Reader) *cobra.Command {
 
 	f := cmd.Flags()
 	f.BoolVarP(&i.clientOnly, "client-only", "c", false, "install local configuration, but skip remote configuration")
+	f.BoolVarP(&i.ingress, "ingress", "i", false, "configure ingress")
```
Comment (bacongobbler) [inline cmd/draft/init.go]: should we change this to --ingress-enabled so it's consistent with draftd's option flag?

Comment (michelleN) [reply cmd/draft/init.go]: yeah let's do that.

File: cmd/draft/init.go (hunk 1 above)
Comment (michelleN) [reply cmd/draft/init.go]: good eye

**Pull request merged**